### ISSUE TITLE
[Bug] `docker-compose up` fails (postgres interactive prompt & forbidden path outside the build context)

### ISSUE THREAD

**Issue by hatarist** [AppFlowy-IO/AppFlowy-Cloud#46]
Looks like I solved both issues myself, but I'm not really familiar with either Docker or Appflowy and also my solutions look like poop, so I don't feel confident at making a pull request, sorry!

Docker server 24.0.5 on Ubuntu 20.04.6

## issue 1: postgres interactive prompt

When trying to run `docker-compose -d`, while building the postgres container, it asks to choose what to do with a `createcluster.conf` file:

```
Configuring postgresql-common
-----------------------------
createcluster.conf: A new version (/tmp/postgresql-common.sK2jGD) of
configuration file /etc/postgresql-common/createcluster.conf is available, but
the version installed currently has been locally modified.
  1. install the package maintainer's version
  2. keep the local version currently installed
  3. show the differences between the versions
  4. show a side-by-side difference between the versions
  5. start a new shell to examine the situation
What do you want to do about modified configuration file createcluster.conf? 
```

Inputting anything doesn't work (docker seems to ignore stdin, regardless of if it's ran with or without the `-d`aemon argument).

I managed to fix this by adding an env var  to the docker-compose.yml's postgres section:
```
version: '3'
services:
  postgres:
    build:
      context: .
      dockerfile: postgres.Dockerfile
    environment:
      - DEBIAN_FRONTEND=noninteractive
      - POSTGRES_USER=${POSTGRES_USER:-postgres}
      - POSTGRES_DB=${POSTGRES_DB:-postgres}
      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-password}
      - POSTGRES_HOST=${POSTGRES_HOST:-postgres}
    ports:
      - 5433:5432
    volumes:
      - ./migrations/before:/docker-entrypoint-initdb.d
```

(and, maybe, by hard-coding the version in docker/postgres.Dockerfile to `FROM postgres:15.4`, but it seems to be the `latest` now anyway)

## issue 2: appflowy_cloud build fails on `COPY .. .`

```
 ---> 5db4a013643c
Step 6/21 : COPY .. .
ERROR: Service 'appflowy_cloud' failed to build: COPY failed: forbidden path outside the build context: .. ()
```

Seems that it was introduced by e2e6d79 (when dockerfiles went to a separate folder), but I'm not sure, it was a week ago, someone should have already noticed that.

https://stackoverflow.com/questions/69615253/copy-failed-forbidden-path-outside-the-build-context-docker-compose

I managed to "fix" it too by moving `docker/Dockerfile` back to the root and changing `COPY ..  .` to `COPY . .` there.

**Comment by speed2exe** [comment]
@hatarist Thanks for filing the issue.

I have tried with spinning up Ubuntu 22.04 in AWS and ran `docker-compose up -d` at the root of the repo after installing docker-compose.

Issue 1: I was unable to reproduce the first issue, I am guessing it happens because you might be using desktop version of docker?

Issue 2: I was able to reproduce the second issue. I realised that the docker-compose version was different and probably caused that. My main machine was running 2.20.3 while the one in the AWS Ubuntu is at 1.29.0. We will make a fix on this.
